frontend/components/CellInput.js

```diff
@@ -56,74 +60,227 @@ import { bracketMatching } from "./CellInput/block_matcher_plugin.js"
+        {
+            tag: tags.standard(tags.variableName),
+            color: "var(--cm-builtin-color)",
+            fontWeight: 700,
+        },
+
+        { tag: tags.bool, color: "var(--cm-builtin-color)", fontWeight: 700 },
+
+        { tag: tags.keyword, color: "var(--cm-keyword-color)" },
+        { tag: tags.comment, color: "var(--cm-comment-color)", fontStyle: "italic" },
+        { tag: tags.atom, color: "var(--cm-atom-color)" },
+        { tag: tags.number, color: "var(--cm-number-color)" },
+        // { tag: tags.property, color: "#48b685" },
+        // { tag: tags.attribute, color: "#48b685" },
+        { tag: tags.keyword, color: "var(--cm-keyword-color)" },
+        { tag: tags.string, color: "var(--cm-string-color)" },
+        { tag: tags.variableName, color: "var(--cm-var-color)", fontWeight: 700 },
+        // { tag: tags.variable2, color: "#06b6ef" },
+        { tag: tags.typeName, color: "var(--cm-type-color)", fontStyle: "italic" },
+        { tag: tags.typeOperator, color: "var(--cm-type-color)", fontStyle: "italic" },
+        { tag: tags.bracket, color: "var(--cm-bracket-color)" },
+        { tag: tags.brace, color: "var(--cm-bracket-color)" },
+        { tag: tags.tagName, color: "var(--cm-tag-color)" },
+        { tag: tags.link, color: "var(--cm-link-color)" },
+        {
+            tag: tags.invalid,
+            color: "var(--cm-error-color)",
+            background: "var(--cm-error-bg-color)",
+        },
+
+        // PYTHON SPECIFIC
+    ],
     {
-        tag: tags.standard(tags.variableName),
-        color: "var(--cm-builtin-color)",
-        fontWeight: 700,
-    },
-
-    { tag: tags.bool, color: "var(--cm-builtin-color)", fontWeight: 700 },
-
-    { tag: tags.keyword, color: "var(--cm-keyword-color)" },
-    { tag: tags.comment, color: "var(--cm-comment-color)", fontStyle: "italic" },
-    { tag: tags.atom, color: "var(--cm-atom-color)" },
-    { tag: tags.number, color: "var(--cm-number-color)" },
-    // { tag: tags.property, color: "#48b685" },
-    // { tag: tags.attribute, color: "#48b685" },
-    { tag: tags.keyword, color: "var(--cm-keyword-color)" },
-    { tag: tags.string, color: "var(--cm-string-color)" },
-    { tag: tags.variableName, color: "var(--cm-var-color)", fontWeight: 700 },
-    // { tag: tags.variable2, color: "#06b6ef" },
-    { tag: tags.typeName, color: "var(--cm-type-color)", fontStyle: "italic" },
-    { tag: tags.typeOperator, color: "var(--cm-type-color)", fontStyle: "italic" },
-    { tag: tags.bracket, color: "var(--cm-bracket-color)" },
-    { tag: tags.brace, color: "var(--cm-bracket-color)" },
-    { tag: tags.tagName, color: "var(--cm-tag-color)" },
-    { tag: tags.link, color: "var(--cm-link-color)" },
+        scope: pythonLanguage.topNode,
+        all: {
+            color: "var(--cm-editor-text-color)",
+            filter: `contrast(0.5)`,
+        },
+    }
+)
+
+export const pluto_syntax_colors_css = HighlightStyle.define(
```

Comment:
We have a lot of `#rrggbb` colors for the CSS syntax, can those all be CSS variables? Maybe just reuse random vars from Julia, even if they are not a sematic match

Reply:
I'll give CSS another try